src/pages/office/cat-cpc/components/combination/combination.html

```diff
@@ -0,0 +1,17 @@
+<ion-row class="mes-validated-row mes-row-separator" id="combination-card">
+  <div class="validation-bar"></div>
+  <ion-col class="component-label" col-32 align-self-center>
+    <label>Combination</label>
+  </ion-col>
+  <ion-col align-self-center padding-left>
+    <ion-row class="spacing-row"></ion-row>
+    <ion-row>
+      <ion-col align-self-center>
+        <label id="combination-label">{{combination}} - {{combinationAdditionalText}}</label>
```

Comment:
think you need to handle combinations with no additional text or we will just get the '-' suffixed?

Reply:
addressed